client/src/components/OverviewProfileCard.js

```diff
@@ -1,85 +1,109 @@
-import React, { useContext } from 'react';
-import { Card } from 'react-bootstrap';
-import styled from 'styled-components';
-import ViewSkills from '../components/ViewSkills';
-import StyledButton from '../constant/StyledButton';
-import avatar from '../assets/icons/avatar.svg';
-import { useHistory } from 'react-router-dom';
+import React, { useContext } from "react";
+import { Card } from "react-bootstrap";
+import styled from "styled-components";
+import ViewSkills from "../components/ViewSkills";
+import StyledButton from "../constant/StyledButton";
+import { useHistory } from "react-router-dom";
+import GitHubIcon from "./GitHubIconIcon.js";
+import LinkedinIcon from "./LinkedinIcon";
+import CvIcon from "./CvIcon";
+import EmailIcon from "./EmailIcon";
 
+const OverviewProfileCard = ({ profile, getProfile}) => {
+  let history = useHistory();
 
-const OverviewProfileCard = ({ profile, getProfile }) => {
-	let history = useHistory()
-
-	const handleClick = (name)=>{
-		getProfile(name)
-		history.push('/viewdetail')
-	};
-
-	return (
-		<CardContainer>
-			{/* <Img variant="top" src={profile.img||avatar} /> */}
-			<Img />
-			<CardBody>
-				<CardTitle>{profile.first_name} {profile.last_name}</CardTitle>
-				<CardText>
-				Dolorem ipsum quia dolor sit amet, excepturi sint occaecati
-				</CardText>
-				{/* <ViewSkills skills={profile.skills} /> */}
-				<StyledButton name='View Profile' handleClick={()=>handleClick(`${profile.user_name}`)} />
-			</CardBody>
-		</CardContainer>
-	);
+  const handleClick = async (id) => {
+    await getProfile(id);
+    history.push(`/viewdetail/${profile.first_name}`);
+  };
+  return (
+    <CardContainer>
+      <Img src={profile.avatar_url}/>
+      <CardBody>
+        <CardTitle>
+          {profile.first_name} {profile.surname}
+        </CardTitle>
+        <SubContainer>
+          {profile.about_me&&<CardText>{profile.about_me}</CardText>}
+        </SubContainer>
+        <SubContainer>
+         {profile.about_me&&<CardText>{profile.interest}</CardText>}
+        </SubContainer>
+        {/* {profile.skills&&<ViewSkills skills={profile.skills} />} */}
+        <IconContainer>
+          <GitHubIcon gitHubLink={profile.github_link}></GitHubIcon>
+          <LinkedinIcon linkedinLink={profile.linkedin_link}></LinkedinIcon>
+          <CvIcon></CvIcon>
+          <EmailIcon></EmailIcon>
```

Comment:
If your element doesn't have a child, you can simplify them by making the tags self-closing:
```suggestion
          <GitHubIcon gitHubLink={profile.github_link}/>
          <LinkedinIcon linkedinLink={profile.linkedin_link}/>
          <CvIcon/>
          <EmailIcon/>
```